### DISCUSSION TITLE
BLUECFD CORE MAC DOWNLOAD

### DISCUSSION THREAD

**Discussion by archistudentusingmac** [blueCFD/Core#248] [General]
Is there a way to download BlueCFD Core on a macos intel using Docker and XQuartz? I already have OpenFoam and Paraview downloaded.

**Comment by wyldckat** [comment]
I'm not fully understanding your question... blueCFD-Core was designed to work on Microsoft Windows.

Specially because it's borderline impossible (at least without Cygwin) to compile OpenFOAM.org or OpenFOAM.com directly on Windows, without any modifications to the file structure.

On MacOS, you should be able to:
- Build/install OpenFOAM.com without too many issues: https://develop.openfoam.com/Development/openfoam/-/wikis/building#darwin-mac-os
- And OpenFOAM.org with patches: https://openfoamwiki.net/index.php/Installation/Mac_OS
